client/src/containers/ScoreContainer/package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Also please this remove `package.json` file since we are now using `index.js`. Both files serve the same thing.